<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMainWindow</class>
 <widget class="QMainWindow" name="DMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dolphin</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QStackedWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>21</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="mnuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionBrowse"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="mnuEmulation">
    <property name="title">
     <string>E&amp;mulation</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="mnuOptions">
    <property name="title">
     <string>Optio&amp;ns</string>
    </property>
   </widget>
   <widget class="QMenu" name="mnuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="mnuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="mnuListStyle">
     <property name="title">
      <string>&amp;List style</string>
     </property>
     <addaction name="actionListView"/>
     <addaction name="actionTreeView"/>
     <addaction name="actionGridView"/>
     <addaction name="actionIconView"/>
    </widget>
    <addaction name="mnuListStyle"/>
   </widget>
   <widget class="QMenu" name="mnuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWebsite"/>
    <addaction name="actionOnlineDocs"/>
    <addaction name="actionGitHub"/>
    <addaction name="separator"/>
    <addaction name="actionSystemInfo"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="mnuMovie">
    <property name="title">
     <string>Movie</string>
    </property>
   </widget>
   <addaction name="mnuFile"/>
   <addaction name="mnuView"/>
   <addaction name="mnuEmulation"/>
   <addaction name="mnuMovie"/>
   <addaction name="mnuOptions"/>
   <addaction name="mnuTools"/>
   <addaction name="mnuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionWebsite">
   <property name="text">
    <string>&amp;Website</string>
   </property>
  </action>
  <action name="actionOnlineDocs">
   <property name="text">
    <string>&amp;Online Documentation</string>
   </property>
  </action>
  <action name="actionGitHub">
   <property name="text">
    <string>&amp;GitHub Repository</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSystemInfo">
   <property name="text">
    <string>&amp;System Information</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionListView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;List view</string>
   </property>
  </action>
  <action name="actionTreeView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Tree view</string>
   </property>
  </action>
  <action name="actionGridView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Grid view</string>
   </property>
  </action>
  <action name="actionIconView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Icon view</string>
   </property>
  </action>
  <action name="actionBrowse">
   <property name="text">
    <string>&amp;Browse for ISOs...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
